exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,84 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "A basket containing only a single book should total to single book price.",
+            "basket": [1],
+            "expected": 8.0
+         },
+         {
+	    "description": "A basket containing two of the same book should total to single book price x2.",
+            "basket": [2, 2],
+            "expected": 16.0
+         },
+         {
+            "description": "Should be able to cope with an empty basket.",
+            "basket": [],
+            "expected": 0.0
+         },
+         {
+	    "description": "Should arrive at best discount and correct total for a basket containing two different books.",
+            "basket": [1, 2],
+            "expected": 15.2
+         },
+         {
+	    "description": "Should arrive at best discount and correct total for a basket containing three different books.",
+            "basket": [1, 2, 3],
+            "expected": 21.60
+         },
+         {
+	    "description": "Should arrive at best discount and correct total for a basket containing four different books.",
+            "basket": [1, 2, 3, 4],
+            "expected": 25.60
+         },
+         {
+	    "description": "Should arrive at best discount and correct total for a basket containing five different books.",
+            "basket": [1, 2, 3, 4, 5],
+            "expected": 30.00
+         },
+         {
+	    "description": [
+		             "Should arrive at best discount and correct total for a basket containing 8 books.",
+		             "The basket will consist of two each of the first 3 books and one each of books 4 and 5."
```

Comment:
@petertseng when/how is the canonical data presented to the student?  I think it would help me if I knew the context of when each of the three (description.md, canonical-data.json, and metadata.yml) files is used.